src/test/java/uk/gov/companieshouse/authcodenotification/controller/AuthCodeNotificationControllerTest.java

```diff
@@ -1,16 +1,40 @@
 package uk.gov.companieshouse.authcodenotification.controller;
 
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.http.ResponseEntity;
+import uk.gov.companieshouse.authcodenotification.exception.ServiceException;
+import uk.gov.companieshouse.authcodenotification.service.AuthCodeNotificationService;
+
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.doThrow;
 
+@ExtendWith(MockitoExtension.class)
 class AuthCodeNotificationControllerTest {
 
-    @Test
-    void testSendEmailReturnsSuccess (){
+    private static final String REQUEST_ID = "abc";
+
+    private static final String COMPANY_NUMBER = "OE123456";
+
+    @InjectMocks
+    private AuthCodeNotificationController controller;
+
+    @Mock
+    private AuthCodeNotificationService authCodeNotificationService;
 
-        AuthCodeNotificationController controller = new AuthCodeNotificationController();
-        ResponseEntity<Object> responseEntity = controller.sendEmail("abc","012345657");
+    @Test
+    void testSendEmailReturnsSuccess() {
+        ResponseEntity<Object> responseEntity = controller.sendEmail(REQUEST_ID ,COMPANY_NUMBER);
         assertEquals( 200, responseEntity.getStatusCode().value() );
     }
+
+    @Test
+    void testSendEmailException() throws ServiceException{
```

Comment:
Suggest renaming this to 'testSendEmailReturnsInternalServerErrorWhenServiceCallFails'.

Reply:
Method name changed.

Reply:
Thanks